### ISSUE TITLE
Make default tailwind.css configurable

### ISSUE THREAD

**Issue by kotgakapainor** [brandonpittman/gridsome-plugin-tailwindcss#62]
##Current State

As using the tailwind plugin api is tiring and an unnecessary layer of abstraction , I propose to leave it to the user if they want to use `CSS in JS` or `CSS/SCSS` etc.

Its also stated at [tailwind](https://tailwindcss.com/docs/adding-base-styles/#when-to-use-a-plugin):

> In general, it's simpler to add base styles to your project in CSS than it is to write a plugin.

## Solution

Remove the line `import "tailwindcss/tailwind.css";` in `gridsome.client.js`, Then the user can import their own CSS file and modify it like this (copy/pasted from [tailwindcss.com](https://tailwindcss.com/docs/adding-base-styles/#using-css)):

```
@tailwind base;

h1 {
  @apply text-2xl;
}
h2 {
  @apply text-xl;
}
h3 {
  @apply text-lg;
}
a {
  @apply text-blue-600 underline;
}

@tailwind components;

@tailwind utilities;
```

**Comment by brandonpittman** [comment]
Nope.

**Comment by kotgakapainor** [comment]
Is there a reason or do you just dont want to?

I dont want so sound rude, but a one word answer looks lazy.

**Comment by brandonpittman** [comment]
@kotgakapainor I made it clear in the README that the plugin doesn't support that style of usage and I'm dead set against using Tailwind that way. If you want to use Tailwind that way, I'd suggest you just use it as a normal PostCSS plugin (as detailed on the Tailwind homepage).

I thought about writing a longer reply the first time, but since the first thought that popped into my head was "Nope", I went with it. 🤣
